pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -55,7 +57,13 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
-var LegacyAPIGroupPrefixes = sets.NewString(genericapiserver.DefaultLegacyAPIPrefix, api.Prefix, api.LegacyPrefix)
+var (
+	LegacyAPIGroupPrefixes = sets.NewString(genericapiserver.DefaultLegacyAPIPrefix, api.Prefix, api.LegacyPrefix)
+
+	// request paths that match this regular expression will be treated as long running
+	// and not subjected to the default server timeout.
+	originLongRunningEndpointsRE = "(/|^)buildconfigs/.*?/instantiatebinary$"
```

Comment:
I think it's probably unnecessary in this case, but i generally try to default to non-greedy matching.  tends to have less surprises.